skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/ui/SkaffoldProfilesComboBox.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.container.tools.skaffold.run.ui
+
+import com.google.common.annotations.VisibleForTesting
+import com.google.container.tools.skaffold.SkaffoldYamlConfiguration
+import com.google.container.tools.skaffold.message
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.vfs.VirtualFile
+import javax.swing.DefaultComboBoxModel
+import javax.swing.JComboBox
+
+/**
+ * Combobox for selecting Skaffold profiles. Default profile is assumed to be always present for a
```

Comment:
by "assumed to be always present" do you mean that it is implicitly there always (even when no profile is defined in the yaml)?

Reply:
Right, if Skaffold file is correct, it is itself presents a "default profile". Perhaps I can slightly rephrase to make it more obvious.